src/screens/main/rank/components/rankUser/RankUser.tsx

```
@@ -0,0 +1,19 @@
+import Typography from "@/components/common/typography/Typography";
+import * as S from "./style"
+import { Image } from "react-native";
+import { RankUserProps } from "@/types/main";
+
+export default function RankUser({ rank, profileImage, profileName, bookPoint }: RankUserProps) {
+  return(
+    <S.Container>
+      <S.LeftBox>
+        <Typography font='semiBold20' color='defaultBlack' children={rank.toString()} />
```

Biome (2.4.4)

[error] Avoid passing children using a prop

(lint/correctness/noChildrenProp)